src/core/variable.jl

```diff
@@ -81,9 +87,11 @@ function variable_mc_voltage_imaginary(pm::_PMs.AbstractPowerModel; nw::Int=pm.c
     )
 
     if bounded
-        for (i,bus) in _PMs.ref(pm, nw, :bus), c in cnds
-            JuMP.set_lower_bound(vi[i][c], -bus["vmax"][c])
-            JuMP.set_upper_bound(vi[i][c],  bus["vmax"][c])
+        for (i,bus) in _PMs.ref(pm, nw, :bus)
+            if haskey(bus, "vmax")
+                JuMP.set_lower_bound.(vi[i], -bus["vmax"])
+                JuMP.set_upper_bound.(vi[i],  bus["vmax"])
```

Comment:
should we instead be using the internal `set_lower_bound` and `set_upper_bound` here?